photon-lib/src/main/native/include/photon/simulation/PhotonCameraSim.h

```diff
@@ -51,6 +51,8 @@ class PhotonCameraSim {
  public:
   explicit PhotonCameraSim(PhotonCamera* camera);
   PhotonCameraSim(PhotonCamera* camera, const SimCameraProperties& props);
+  PhotonCameraSim(PhotonCamera* camera, const SimCameraProperties& props,
+                  const frc::AprilTagFieldLayout& tagLayout);
```

Comment:
Consider using a default argument instead of an overloaded constructor. Not sure which is more idiomatic.